pkg/scheduler/apis/config/v1beta1/conversion.go

```diff
@@ -107,3 +107,19 @@ func convertToExternalPluginConfigArgs(out *v1beta1.KubeSchedulerConfiguration)
 	}
 	return nil
 }
+
+func Convert_config_Extender_To_v1beta1_Extender(in *config.Extender, out *v1beta1.Extender, s conversion.Scope) error {
+	if err := autoConvert_config_Extender_To_v1beta1_Extender(in, out, s); err != nil {
+		return err
+	}
+	out.HTTPTimeout.Duration = in.HTTPTimeout
```

Comment:
@liggitt why is this not handled by the generated conversion? Is it an anti-pattern to use time.Duration as the internal type?

Reply:
Yep, I had a feeling that it should be handled there. 

Other parts of k8s use `metav1.Duration` in internal type which offers no advantages over `time.Duration` apart of autoconvert.

Reply:
use metav1.Duration in API type fields